frontend/src/components/MiniCal/MiniCal.tsx

```diff
@@ -33,6 +33,7 @@ const Icon = () => (
 
 const MiniCal = () => {
   const { curDate, setCurDate } = useDate();
+  const datePickerRef = React.useRef<any>(null);
```

Comment:
We can extend the types via

```
declare module 'react-datepicker' {
  export default class ReactDatePicker {
    setPreSelection?: (date: Date) => void;
  }
}
```

to be able to do

```suggestion
  const datePickerRef = React.useRef<DatePicker>(null);
```

Reply:
imo we don't want to introduce more `any`s into the system if we don't have to